Update documentation for Admin UI SDK 2.3.0 release (#332)
AdobeDocs/commerce-extensibility: src/pages/admin-ui-sdk/release-notes.md

# Adobe Commerce Admin UI SDK release notes

## Version 2.3.0

### Release date

March 14, 2025

### Enhancements

> * Rename Staging test mode to Sandbox. <!--- CEXT-4072  -->

Reviewer: ```suggestion
* Renamed Staging **Test mode** to Sandbox. <!--- CEXT-4072  -->
```

> * Align statuses label with App Registry. <!--- CEXT-4071  -->

Reviewer: ```suggestion
* Aligned the **App Status** label with the App Registry. <!--- CEXT-4071  -->
```

> * Add a tooltip to clarify the testing mode options. <!--- CEXT-4134  -->

Reviewer: ```suggestion
* Added a tooltip to clarify the **Test mode** options. <!--- CEXT-4134  -->
```

### Bug fixes

> * Fix REST API order retrieval error when Admin UI SDK custom fees module is enabled. <!--- CEXT-4348  -->
> 
> * Fix issue with timeout when contacting App Registry. <!--- CEXT-4205  -->

Reviewer: ```suggestion
* Fixed a REST API order retrieval error that could occur when the Admin UI SDK custom fees module was enabled. <!--- CEXT-4348  -->

* Fixed a timeout issue that could occur when contacting the App Registry. <!--- CEXT-4205  -->
```
